index.js

```diff
@@ -101,41 +112,63 @@ async function exec(watchItem, screenshotPath)
         return false;
     }
 
-    let htmlContent = await item.innerHTML();
+    let htmlContent = await page.evaluate(el => el.innerHTML, item);
 
-    //screenshot on change
+    // Screenshot on change
     if (watchItem.lastContent && htmlContent != watchItem.lastContent)
         await item.screenshot({path: screenshotPath});
 
     await browser.close();
 
-    //change detected
+    // Change detected
     if (watchItem.lastContent && htmlContent != watchItem.lastContent)
     {
-        watchItem.changeDetected = true;
         log('change detected', colors.rainbow);
+
+        if(watchItem.minValue) {
+
+            if(parseInt(htmlContent.replace(/,/,"")) > parseInt(watchItem.minValue)) {
```

Comment:
this replacement does not apply for generic values, if you have a german foramtted number like: `10,10` --> `1000`